src/component/mxgraph/GraphCellUpdater.ts

```diff
@@ -108,12 +109,31 @@ export default class GraphCellUpdater {
           font.isStrikeThrough !== undefined &&
             (cellStyle = mxgraph.mxUtils.setStyleFlag(cellStyle, mxgraph.mxConstants.STYLE_FONTSTYLE, mxgraph.mxConstants.FONT_STRIKETHROUGH, font.isStrikeThrough));
 
-          // TODO To uncomment when we implement the Opacity in global/background/font/stroke
+          // TODO To uncomment when we implement the Opacity in global/fill/font/stroke
           // font.opacity && (cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_TEXT_OPACITY, font.opacity));
         }
 
+        if (isShapeStyleUpdate(styleUpdate)) {
+          const fill = styleUpdate.fill;
+          if (fill) {
+            if (fill.color) {
+              cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_FILLCOLOR, fill.color);
+              if (cellStyle.includes(ShapeBpmnElementKind.POOL) || cellStyle.includes(ShapeBpmnElementKind.LANE)) {
```

Comment:
**thought:** for myself
Check if don't have the same need elsewhere in the code. If so, use the same function to handle this

Reply:
Only used in integration tests with an uggly implementation. This one is better.
Decision with @csouchet keep as it is in the PR.

https://github.com/process-analytics/bpmn-visualization-js/blob/e6317b675bbed16f399dd0cba1650e65a61380c7/test/integration/helpers/model-expect.ts#L199-L202